server/guardrails/input_rail.py

```
@@ -172,6 +180,17 @@ def _build_rails_sync():
     return LLMRails(config=rails_config, llm=_build_llm())
 
 
+def prewarm_rails_sync() -> None:
+    """Sync init for Celery worker prewarm — thread-safe with async lazy init."""
+    global _rails_instance
```

Ruff (0.15.17)

[warning] Using the global statement to update `_rails_instance` is discouraged

(PLW0603)